db/migrate/20170716043257_create_chesspieces.rb

```diff
@@ -0,0 +1,8 @@
+class CreateChesspieces < ActiveRecord::Migration[5.0]
+  def change
+    create_table :chesspieces do |t|
```

Comment:
We don't need to setup any database tables yet. Can remove this for now.

(will be doing this migrations once the Plan is done towards the end of the week)